.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: node_js
+addons:
+    code_climate:
+        repo_token: $CODECLIMATE_REPO_TOKEN
```

Comment:
I set the repo token in travis-ci using the `CODECLIMATE_REPO_TOKEN` env var. CodeClimate only checks the master branch though. So things will only work once the PR is merged.